### PR TITLE
1.0.0 - Plugin Launch

### LINKED ISSUE #2: Error on plugin activation
The "verify_plugin_dependencies" function is not detecting the dependencies properly causing the plugin not begin activated even if it has all dependencies.

- [x] Simplify verification method of existence of the the LifterLMS plugin.

### PR DESCRIPTION
**Pull request by FabricioQueirozLkn**
## Changelog

Lançamento do plugin.

### v1.0.0 - 25/07/23

- Plugin launch.

## Conferir
- [x] Atualizar versão hardcoded
- [x] Atualizar README com notas
- [x] Atualizar CHANGELOG

### CODE REVIEW

File: README.txt
Comment (emanuellopess) [inline README.txt]: @FabricioLinkNacional Could you put a link to the Lifter LMS plugin? In the "[..] on LifterLMS plugin, [..]" put a markdown link to here: https://wordpress.org/plugins/lifterlms/

File: admin/index.php
Comment (emanuellopess) [inline admin/index.php:1]: @FabricioLinkNacional This should follow the pattern "silence is golden", like the example below:
```php
<?php

// Silence is golden
```

File: admin/lkn-payment-banking-slip-pix-for-lifterlms-slip-settings.php
Comment (emanuellopess) [inline admin/lkn-payment-banking-slip-pix-for-lifterlms-slip-settings.php]: @FabricioLinkNacional Please verify you linting extension, you need to format this file

File: includes/class-lkn-payment-banking-slip-pix-for-lifterlms-helper.php
Comment (emanuellopess) [inline includes/class-lkn-payment-banking-slip-pix-for-lifterlms-helper.php]: @FabricioLinkNacional Needs sanitization, please use [this function](https://developer.wordpress.org/reference/functions/sanitize_url/) to sanitize the $page_url

File: includes/class-lkn-payment-banking-slip-pix-for-lifterlms.php
Comment (emanuellopess) [inline includes/class-lkn-payment-banking-slip-pix-for-lifterlms.php]: @FabricioLinkNacional This function needs the parameter $gateways to work, this function is called in a filter, it doesn't have args but it has a parameter, see more [here](https://developer.wordpress.org/reference/functions/apply_filters/)

It actually deletes all gateways except for the Banking Slip and PIX.

File: public/js/lkn-payment-banking-slip-pix-for-lifterlms-public.js
Comment (emanuellopess) [inline public/js/lkn-payment-banking-slip-pix-for-lifterlms-public.js]: @FabricioLinkNacional Please delete all unused code

File: public/class-lkn-payment-banking-slip-pix-for-lifterlms-public.php
Comment (emanuellopess) [inline public/class-lkn-payment-banking-slip-pix-for-lifterlms-public.php]: @FabricioLinkNacional Add the function '[localize_script](https://developer.wordpress.org/reference/functions/wp_localize_script/)' to load the string translations

File: public/js/lkn-payment-banking-slip-pix-for-lifterlms-public.js
Comment (emanuellopess) [inline public/js/lkn-payment-banking-slip-pix-for-lifterlms-public.js]: @FabricioLinkNacional Needs translation

File: public/js/lkn-payment-banking-slip-pix-for-lifterlms-public.js
Comment (emanuellopess) [inline public/js/lkn-payment-banking-slip-pix-for-lifterlms-public.js]: @FabricioLinkNacional Needs translation

File: admin/class-lkn-payment-banking-slip-pix-for-lifterlms-admin.php
Comment (emanuellopess) [inline admin/class-lkn-payment-banking-slip-pix-for-lifterlms-admin.php]: @FabricioLinkNacional This is not right, when you are localizing a script you need to register the localization to the script handler, in this case, $this->plugin_name, you are using a unnecessary empty script to load the localization.

**Review by emanuellopess — CHANGES_REQUESTED** [review]
Needs little changes before the WordPress approval, call me if you need more details

File: admin/lkn-payment-banking-slip-pix-for-lifterlms-slip-settings.php
Comment (FabricioQueirozLkn) [reply admin/lkn-payment-banking-slip-pix-for-lifterlms-slip-settings.php]: @emanuellopess It doesn't show up broken like that in my vs code, but i moved it anyway.

File: .vscode/.php-cs-fixer.php
Comment (emanuellopess) [inline .vscode/.php-cs-fixer.php:12]: @FabricioLinkNacional Please view the new linting rules defined in the DEV Normatives

**Review by emanuellopess — APPROVED** [review]

**Comment by emanuellopess** [comment]
@FabricioLinkNacional Adicionar biblioteca de plugin-updater e ajustar o script do workflow

File: .github/workflows/main.yml
Comment (ghost) [inline .github/workflows/main.yml:22]: composer install deve rodar antes do passo "Make plugin root directory with auto-updater".

E no passo "Make plugin root directory with auto-updater", deve ser incluída a pasta vendor.

File: CHANGELOG.md
Comment (ghost) [inline CHANGELOG.md:1]: Mesmo sendo a primeira versão, acho interessante inserir quais features o plugin carrega na sua primeira versão.

File: .github/workflows/main.yml
Comment (FabricioQueirozLkn) [reply .github/workflows/main.yml:22]: @brunoferreiralkn  saquei, então nem preciso fazer o composer install duas vezes né, mas no caso ele vai incluido nos dois comandos de "Make plugin root directory" ? Ou só no com Updater ?

**Review by ghost — CHANGES_REQUESTED** [review]
Falta implementar o workflow corretamente.

File: .github/workflows/main.yml
Comment (ghost) [reply .github/workflows/main.yml:22]: Vai nas duas build e o composer install só precisa rodar uma vez.

**Review by ghost — APPROVED** [review]

**Pull request merged**
